code/index.html

```diff
@@ -1,30 +1,159 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="icon" type="image/png" sizes="16x16" href="images/favicon-16x16.png">
+    <link rel="manifest" href="/site.webmanifest">
+    
+    <title>Rain check</title>
+</head>
+<body>
+    
+  <header>
+    <!--makes it possible to jump to the top-->
+    <div id="top"></div>
+      <nav class="navigation">
+          <a href="index.html">Home</a>
+          <a href="#join"><i class="fas fa-tint"></i>Join us</a>
+          <a href="user.html"><i class="fas fa-sign-in-alt"></i>Sign in</a> 
+          <a href="#faq"><i class="far fa-question-circle">FAQ</i></a>
+          <a href="#contact"><i class="fas fa-envelope-square">Contact us</i></a>   
+      </nav>
+        
+      <div class="hero">
+          <video loop muted autoplay>
+              <source src="videos/production ID_5197762.mp4" type="video/mp4">
+              <source src="videos/production ID_5197762.ogg" type="video/mp4">
+          </video>
+          <div class="caption">
+              <h1>Rain<i class="far fa-check-square"></i></h1>
+              <p><em>We might be called rain check, but we believe in delivering when we said we would</em></p>
```

Comment:
what does the <em> tag do? Havent seen that before.

Reply:
Do you mean the `<em></em>`? It makes the text in it cursive